### DISCUSSION TITLE
No need to convert to .ggml or .gguf anymore!

### DISCUSSION THREAD

**Discussion by leejet** [leejet/stable-diffusion.cpp#102] [General]
In the latest commit, I've implemented the functionality to directly load model weights from ckpt, safetensors, and diffusers models/checkpoints. There's no longer a need to convert to .ggml or .gguf formats. You can now share model weights with your comfyui/sd-webui on your computer. There's no necessity to maintain separate .ggml or .gguf formatted weights for sd.cpp anymore. If you encounter any issues while using this, please let me know.

**Comment by Amin456789** [comment]
u r a genius! thanks! no more converting :D

**Comment by Kreijstal** [comment]
could you port that to llama.cpp or whisper.cpp?
